wonjong/app.js

```diff
@@ -0,0 +1,40 @@
+//// 3rd-party package
+require("dotenv").config();
+const express = require("express");
+const cors = require("cors");
+const logger = require("morgan");
+
+//DB connection
+const { DataSource } = require("typeorm");
+const appDataSource = new DataSource({
+  type: process.env.DB_CONNECTION,
+  host: process.env.DB_HOST,
+  port: process.env.DB_PORT,
+  username: process.env.DB_USERNAME,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+});
+
+appDataSource
+  .initialize()
+  .then(() => {
+    console.log("Data Source has been initialized!");
+  })
+  .catch((err) => {
+    console.err("Initialize Error", err);
+  });
+
+//custom package
+const app = express();
+app.use(logger("combined"));
+app.use(cors());
+app.use(express.json());
+
+//http 접속확인
+app.get("/ping", function (req, res) {
+  res.json({ message: "pong" });
+});
```

Comment:
```suggestion
app.get("/ping", function (req, res) {
  return res.status(200).json({ message: "pong" });
});
```
백엔드 api에서 네트워크 요청에 대한 응답을 보내줄 때, 각 응답에 맞는 상태코드 또한 핸들링 할 수 있습니다. 적절한 상태코드를 추가해주세요!